core/state_processor.go

```diff
@@ -48,7 +48,7 @@ const (
 	recentTime             = 1024 * 3
 	recentDiffLayerTimeout = 5
 	farDiffLayerTimeout    = 2
-	reuseSlotDB            = true // reuse could save state object copy cost
+	reuseSlotDB            = false // reuse could save state object copy cost
```

Comment:
we can enable this by default and delete this option.

Reply:
It will be removed soon, we will make a final decision to remove slot db reuse code.
Even with Parallel 2.0: streaming pipeline, SlotDB reuse may not be necessary.